packages/flutter_tools/lib/src/ios/mac.dart

```diff
@@ -44,7 +45,12 @@ class IOSDeviceNotFoundError implements Exception {
 class IMobileDevice {
   const IMobileDevice();
 
-  bool get isInstalled => exitsHappy(<String>['idevice_id', '-h']);
+  bool get isInstalled {
```

Comment:
Cache this in a private instance field instead of spawning a process each time.

Reply:
Done